src/main.ts

```diff
@@ -208,7 +208,22 @@ function toClosureJS(
   // place of the original source.
   let host = createSourceReplacingCompilerHost(tsickleOutput, ts.createCompilerHost(options));
   program = ts.createProgram(fileNames, options, host);
-  let diagnostics = ts.getPreEmitDiagnostics(program);
+
+  // This used to be
+  //
+  //   let diagnostics = ts.getPreEmitDiagnostics(program);
+  //
+  // but that calls program.getDeclarationDiagnostics, which causes
+  // import errors when the Closure-annotated code gets passed
+  // through.  Until that issue is resolved (#189), we can work
+  // around it by inlining getPreEmitDiagnostics, and removing the
+  // offending line.
+
+  let diagnostics = [
+    ...program.getOptionsDiagnostics(),
+    ...program.getSyntacticDiagnostics(),
+    ...program.getGlobalDiagnostics(),
+  ];
   if (diagnostics.length > 0) {
```

Comment:
Just this one?  Should the other calls to `allDianostics.push` in toClosureJS be preserved?